### PR TITLE
Fixing precondition for rule 1.1.1.2

### PR DESCRIPTION
**Pull request by bgro**
**Overall Review of Changes:**
Fix of precondition for task regarding control 1.1.1.2

**Issue Fixes:**

- #112 

**Enhancements:**
n/a

**How has this been tested?:**
Local test environment.

### CODE REVIEW

File: tasks/prelim.yml
Comment (uk-bolly) [inline tasks/prelim.yml:1]: hi @bgro 

This is great, could we add a when: conditional for the rule as this is the only place this is used for the control 1.1.1.2?

thanks as always

uk-bolly

**Review by uk-bolly — APPROVED** [review]
Brilliant work thank you

File: tasks/prelim.yml [hunk 1]
```diff
@@ -8,6 +8,19 @@
   tags:
       - rule_1.1.1.2
       - always
+  when:
+      - ubtu22cis_rule_1_1_1_2
+
+- name: "PRELIM | Register if squashfs is built into the kernel"
+  ansible.builtin.shell: more /lib/modules/$(uname -r)/modules.builtin | grep  -c "squashfs"
```
Comment (tomi-bigpi) [inline tasks/prelim.yml:15]: FYI: This is causing things to hang when running via Packer/Ansible. Likely due to `more` instead of say `cat`

Comment (bgro) [reply tasks/prelim.yml:15]: @tomi-bigpi:  This PR has already been merged ... could you open an issue about this behavior? Sorry, that I broke something :(

**Pull request merged**